docs/network/host_networking.md

```diff
@@ -0,0 +1,346 @@
+# Host Networking in MicroShift
+
+## Table of Contents
```

Comment:
I thought TOC was generated automatically. Can we skip this section?

Reply:
@ggiguash What wiki format do you use to auto-generate the TOC? I tried the following, but doesn't work:
```
[TOC]
{{TOC left}}
```

Reply:
I think it's generated automatically when viewed in GitHub. 
![image](https://user-images.githubusercontent.com/85498834/205933776-b70d17c2-0a95-478a-916a-5806b3f13324.png)

Reply:
fixed.